examples/tox21/data.py

```diff
@@ -34,7 +34,7 @@ def get_test_file_path(self):
 
     def create_cache_directory(self):
         try:
-            os.makedirs(self.cache_dir)
+            os.makedirs(self.cache_dir, exist_ok=True)
```

Comment:
`exist_ok` is available in Python3 only.

Reply:
Oh I see.